AutoCollection/HttpRequests.ts

```diff
@@ -90,6 +90,8 @@ class AutoCollectHttpRequests {
                     // Set up correlation context
                     const requestParser = new HttpRequestParser(request);
                     const correlationContext = this._generateCorrelationContext(requestParser);
+                    CorrelationContextManager.bindEmitter(request);
```

Comment:
I wonder if we should do this in the auto collector for HTTP dependencies as well.